doc/api.apib

```diff
@@ -174,11 +183,342 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 + Request (application/json)
 
         {
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+        }
```

Comment:
Is there a reason not to allow `enabled` PATCH at component level? Also, `next_natural_trigger` is silently ignored according to the current implementation. We can either support it or explicitly return an error to the user